TKGame/Weapons/WeaponSystem.cs

```diff
@@ -84,27 +84,49 @@ public static void LoadContent(VerticalStackPanel VSP)
 
             // Configure labels
             // Sword Label
+            swordConfig(VSP, true);
+
+            // Spear Label
+            spearConfig(VSP, false);
+
+            // Axe Label
+            axeConfig(VSP, false);
+
+        }
+
+        private static void swordConfig(VerticalStackPanel VSP, bool isActive)
+        {
             swordLabel.Text = swordText;
-            swordLabel.TextColor = WEAPON_TEXT_COLOR;
+            if (isActive)
+                swordLabel.TextColor = Color.Red;
+            else
+                swordLabel.TextColor = WEAPON_TEXT_COLOR;
             swordLabel.Font = weaponFontSystem.GetFont(WEAPON_FONT_SIZE);
             swordLabel.Visible = true;
             VSP.Widgets.Add(swordLabel);
```

Comment:
I don't think we want to just have a list of text on the screen to alert the player of what they have/are using so maybe we should look at some roguelike game weapon/ability graphics so we can move more towards a polished look rather than a demoable and simplistic one